src/ios/CDVLocation.m

```diff
@@ -68,99 +68,95 @@ - (BOOL)isAuthorized
 
     if (authorizationStatusClassPropertyAvailable) {
         NSUInteger authStatus = [CLLocationManager authorizationStatus];
-#ifdef __IPHONE_8_0
-        if ([self.locationManager respondsToSelector:@selector(requestWhenInUseAuthorization)]) {  //iOS 8.0+
-            return (authStatus == kCLAuthorizationStatusAuthorizedWhenInUse) || (authStatus == kCLAuthorizationStatusAuthorizedAlways) || (authStatus == kCLAuthorizationStatusNotDetermined);
-        }
-#endif
```

Comment:
Shouldn't this code be kept since it's being used for iOS 8+ versions?

Reply:
I've added it again, but this is 10 versions below the current iOS so it probably won't be missed.